src/components/elements/tooltip/index.tsx

```diff
@@ -0,0 +1,22 @@
+'use client';
+
+import * as RadixTooltip from '@radix-ui/react-tooltip';
+
+import { StyledTooltipContent } from './tooltip.styles';
+import type { TooltipProps } from './tooltip.types';
+import { Text } from '../text';
+
+export const Tooltip = ({ children, text, colorScheme }: TooltipProps) => {
+  return (
+    <RadixTooltip.Root>
+      <RadixTooltip.Trigger asChild>{children}</RadixTooltip.Trigger>
```

Comment:
Currently, the tooltip appears only on top. We need the possibility to handle the side because [where the tooltip is intended](https://www.figma.com/file/O5411bxKHP2O2OyfW19w6E/HTTP-Toolkit---UI-Design?type=design&node-id=347-6730&mode=design&t=kT7M98IjJQnxXOka-0) to use the position is on the right:

<img width="850" alt="Screenshot 2024-02-26 at 17 22 12" src="https://github.com/bejamas/httptoolkit-website/assets/22168189/8cb111db-a86e-4f0a-ba77-1b51ff2f21e2">

Check the side prop https://www.radix-ui.com/primitives/docs/components/tooltip#content